daemon/src/routes_maker.rs

```diff
@@ -154,6 +154,8 @@ pub async fn post_cfd_action(
     };
 
     result.map_err(|e| {
+        tracing::warn!(order_id=%id, %action, "Processing action failed: {e:#}");
```

Comment:
I think we could do this on a more general level with an [`on_response`](https://docs.rs/rocket/0.5.0-rc.1/rocket/fairing/struct.AdHoc.html#method.on_response) rocket fairing: